src/flatbuffersdata/data.go

```diff
@@ -19,6 +19,7 @@ func MarshalAppInfo(info *newrelic.AppInfo) ([]byte, error) {
 	settingsJSON, _ := json.Marshal(info.Settings)
 	envJSON, _ := json.Marshal(info.Environment)
 	labelsJSON, _ := json.Marshal(info.Labels)
+	metadataJSON, _ := json.Marshal(info.Metadata)
```

Comment:
Is this marshaling the JSON string from the axiom side of things into JSON?  What does that yield?  Just trying to visualize what this does.